tensorflow_addons/optimizers/gradient_accumulator.py

```diff
@@ -108,14 +108,14 @@ def _resource_apply_sparse(self, grad: types.TensorLike, var, indices, apply_sta
         def _apply():
             if "apply_state" in self._optimizer._sparse_apply_args:
                 train_op = self._optimizer._resource_apply_sparse(
-                    accum_gradient.sparse_read(indices),
+                    accum_gradient.read_value(),
```

Comment:
It seems one does not have to call `read_value()` here. Just passing in `accum_gradient` works for me as well.